fenix/app/src/main/res/values/strings.xml

```diff
@@ -610,6 +610,10 @@
     <!-- Error displayed when user attempts to install an add-on from AMO (addons.mozilla.org) that is already installed -->
     <string name="addon_already_installed">Add-on is already installed</string>
 
+    <!-- Add-on process crash dialog to user -->
+    <string name="addon_process_crash_dialog_title">Add-ons are temporarily disabled</string>
+    <string name="addon_process_crash_dialog_message">One or more add-ons stopped working, making your system unstable. Firefox unsuccessfully tried to restart the add-on(s).\n\nAdd-ons won’t be restarted during your current session.\n\nIf this continues to happen, try to remove or disable some add-ons.</string>
```

Comment:
We want to replace `Firefox` here with a parameter %s. This is to avoid having our brand names changed by translators. This isn't formally written down anywhere, but some context can be found in https://github.com/mozilla-mobile/firefox-android/pull/3283#discussion_r1295317110.

After adding a parameter, you will also need to describe what the parameter represents in the localization note.

Reply:
Done!